qt_ui/windows/QLiberationWindow.py

```diff
@@ -367,6 +371,32 @@ def onGameGenerated(self, game: Game):
         self.game = game
         GameUpdateSignal.get_instance().game_loaded.emit(self.game)
 
+    def onEndGame(self, state: TurnState):
```

Comment:
End-game message, closing all windows except main, start new campaign wizard if user chooses to start a new campaign.